.github/workflows/calculate-hash.yml

```diff
@@ -0,0 +1,37 @@
+name: calculate-hashes
```

Comment:
Is this something we should bake into the GH release process / CI builds, and just leave as artifacts there?

Reply:
We could probably have a more automated release process that automatically calculates these hashes and opens a PR for us on release -- I can add that to our nix-agent automated release improvement issue whenever I have access to view issues again. 😅 

(We use these in the nix derivation: https://github.com/kolide/nix-agent/blob/main/kolide-launcher.nix#L13.)